### PR TITLE
Update pods

### PR DESCRIPTION
**Pull request by djbe**
Depends on #619

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

Seems like everything is in order 👍 You did a good job here! 🤝

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -92,6 +92,9 @@
 * Updated to CocoaPods 1.9.0.  
   [David Jennes](https://github.com/djbe)
   [#619](https://github.com/SwiftGen/SwiftGen/pull/619)
+* Updated Pods and Gems dependencies
```
Comment (SwiftGen-Eve) [inline CHANGELOG.md]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/4900d105ba6e710fbc57d0ee405a9ea137dd73f1/CHANGELOG.md#L95"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>

**Comment by AliSoftware** [comment]
@djbe Any idea why in SPM it can't find `Config` and `LogLevel` types when trying to compile the tests? Not sure I see what changed that would explain this


EDIT: Ok nvm. `rake spm:test` has in fact never worked because SPM can't yet test executables (and depend on executables like the `SwiftGen` product), that's nothing new. Previous SPM failure was different on CI and was because `Commander` update in `Package.swift` was not reflecting the update in `Podfile`, which I fixed in recent commit. So all good now!

**Comment by AliSoftware** [comment]
@djbe Is it ok if I un-draft this PR? I'm guessing it was a draft so far because it was based on branch `feature/cocoapods-1.7-beta` and waiting for #619 to be merged, but now that it is and that I've changed base branch back to `develop` that should be good, right?

**Comment by djbe** [comment]
Looks good.

- [x] Add the PR/issue number from #688, as we're already fixing the typo here
- [x] Don't forget to remove the commented out CI bit if it's no longer needed (libxml2 stuff)

**Comment by djbe** [comment]
On another note: if the Kanna (libxml) changes work as described for everyone, then we'll need to update our installation notes in the Readme accordingly.

Is the min. Xcode requirement something for us (who build the binaries), or for the end user? If I remember correctly, some brew users install from source, right? Will everything keep working for them?

**Comment by AliSoftware** [comment]
> Is the min. Xcode requirement something for us (who build the binaries), or for the end user? If I remember correctly, some brew users install from source, right? Will everything keep working for them?

That's a good question. My only source about "libxml doesn't need to be installed anymore starting Xcode 11.4" is what I read in [the GH Release notes of Kanna 5.2.0](https://github.com/tid-kijyun/Kanna/releases/tag/5.2.0). We might need to investigate more about this.

I say we merge this PR as is keep that doc update for later though since that'd need to first ensure in which cases that's still needed (only for developers of SwiftGen wanting to build it from source and build Kanna as dependency? Or also at runtime for users of the pre-compiled SwiftGen binary depending on various dylibs including Kanna? If the latter, is the fact that Xcode 11.4 is installed, even if it's not the one used and `xcode-select`'d, enough for libxml to be found at runtime by dyld? etc)

**Review by AliSoftware — APPROVED** [review]

**Comment by AliSoftware** [comment]
🎉

**Pull request merged**